cloudsync/sync/manager.py

```diff
@@ -1396,6 +1392,8 @@ def embrace_change(self, sync, changed, synced):  # pylint: disable=too-many-ret
                     # force the trash to sync instead
                     # removing this flakes test: folder_conflicts_del shuffled/oid_is_path version
                     # also breaks test_folder_del_loop
+                    #
+                    # NOTE: above-mentioned tests no longer fail with this if-block removed
```

Comment:
Is the older comment still true at all? If not, perhaps it makes sense to remove it instead of negating it with a new comment...

Reply:
Will change this to a TODO -- AFAICT this block is not hit by any existing test, it would be good try to create/modify tests to hit it before deciding to delete it entirely. As this is unrelated to nested syncs and I don't want to hold up this PR much longer, will come back to it later.

Reply:
Please create a ticket and reference the ticket number in the TODO, thx